openmc/mgxs/mgxs.py

```diff
@@ -1934,19 +1934,38 @@ def print_xs(self, subdomains='all', nuclides='all', xs_type='macro'):
 
 
 class TotalXS(MGXS):
-    """A total multi-group cross section.
+    r"""A total multi-group cross section.
 
     This class can be used for both OpenMC input generation and tally data
     post-processing to compute spatially-homogenized and energy-integrated
-    multi-group cross sections for multi-group neutronics calculations.
+    multi-group total cross sections for multi-group neutronics calculations. At
+    a minimum, one needs to set the :attr:`TotalXS.energy_groups` and
+    :attr:`TotalXS.domain` properties. Tallies for the flux and appropriate
+    reaction rates over the specified domain are generated automatically via the
+    :attr:`TotalXS.tallies` property, which can then be appended to a
+    :class:`openmc.Tallies` instance.
+
+    For post-processing, the :meth:`MGXS.load_from_statepoint` will pull in the
+    necessary data to compute multi-group cross sections from a
+    :class:`openmc.StatePoint` instance. The derived multi-group cross section
+    can then be obtained from the :attr:`TotalXS.xs_tally` property.
+
+    For a spatial domain :math:`D` and energy group :math:`[E_g,E_{g-1}]`, the
+    total cross section is calculated as:
+
+    .. math::
+
+       \frac{\int_{r \in D} dr \int_{4\pi} d\Omega \int_{E_g}^{E_{g-1}} dE \;
```

Comment:
Ditto this comment for all of the cases. This may seem unnecessary, but I think it is important and something that makes our calculation approach with MC unique. Plus, it is important to remind users that we do make this approximation as it can induce very large errors in downstream MG calculations even for simple geometries.